pehlaschoollibrary/proj.android-studio/app/src/com/maq/pehlaschool/library/DownloadExpansionFile.java

```diff
@@ -24,8 +24,6 @@
 import com.google.android.vending.expansion.downloader.IDownloaderClient;
 import com.google.android.vending.expansion.downloader.IDownloaderService;
 import com.google.android.vending.expansion.downloader.IStub;
-import com.maq.pehlaschool.library.MainActivity;
-import com.maq.pehlaschool.library.R;
```

Comment:
Why are these imports removed?

Reply:
It was removed while refracting the code when moving the DownloadExpansion.java from the kitkitschool folder to the com.maq.pehlaschool.library and if the classes are in same folder then the classes can used without the import statements

![image](https://user-images.githubusercontent.com/24955098/63491802-c98c3d00-c4d5-11e9-8e93-1d7de56cc007.png)